tests/test_free_energy.py

```diff
@@ -565,7 +565,7 @@ def test_estimate_free_energy_bar_with_energy_overflow():
     assert np.isfinite(result_with_nan.dG_err)
 
     u_kln_with_inf = np.array(u_kln)
-    u_kln_with_inf[0, 1, 10] = np.inf
+    u_kln_with_inf[1, 0, 10] = np.inf
```

Comment:
curiosity: does this change (and the one on L553) make any difference?

Reply:
No it doesn't. I just wanted to be consistent with what the previous code was doing.